common/views.py

```
@@ -92,6 +93,47 @@ def user_login(request_data, model_class, serializer_class, user_type_name='user
 
+def verify_user(request, email, model_class, user_type='user'):
+    otp = request.data.get('otp')
+    if request.META.get('HTTP_X_FORWARDED_FOR'): ip = request.META.get('HTTP_X_FORWARDED_FOR').split(',')[0]
+    else: ip = request.META.get('REMOTE_ADDR')
+    if not (email and otp): return Response({'error': 'Email and OTP are required'}, status=status.HTTP_400_BAD_REQUEST)
+    if is_ip_throttled(ip): return Response({'error': 'Too many requests from this IP. Try again.'}, status=status.HTTP_429_TOO_MANY_REQUESTS)
+    track_ip_request(ip)
+    if is_otp_brute_forced(email):
+        backoff = increase_backoff(email)
+        return Response({'error': f'Too many OTP attempts. Try again after {backoff} seconds.'}, status=status.HTTP_429_TOO_MANY_REQUESTS)
+    if not verify_otp(email, otp): return Response({'error': 'Invalid or expired OTP'}, status=status.HTTP_400_BAD_REQUEST)
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)